R/lasso-deployment.R

```diff
@@ -7,16 +7,14 @@
 #' \item Push these predictions to SQL Server
 #' }
 #' @docType class
-#' @usage LassoDeployment(type, df, grainCol, testWindowCol, predictedCol, 
-#' impute, debug)
+#' @usage LassoDeployment(type, df, grainCol, predictedCol, impute, debug, cores, modelName)
```

Comment:
On line 20, let the user know that impute uses the values from the model that was saved in the development step.